draft/implementation-notes/index.html

```diff
@@ -602,7 +602,154 @@ <h2>Forward Delta</h2>
         </li>
       </ul>
     </section>
+    
+    <section>
+      <h2>Guidance on Updating OCFL Object Versions</h2>
+      <p>
+      The OCFL is designed to be a specification that covers objects at rest and consequently
+      does not specify in detail update and file locking mechanisms since these are implementation
+      dependent features. Nevertheless, this section includes some basic guidance on how to 
+      update OCFL objects in a manner that tries to ensure that updates are limited to a single  
+      transaction and that failures are detectable and recoverable.
+      </p>
+    
+      <section>
+        <h2>Temporary Names</h2>
+        <p>  
+        While an OCFL object is being updated, and a new version is created, files and folders 
+        being written to should have temporary names that are not meaningful to OCFL parsers.
+        The top level inventory should also be updated last. This ensures that all the files 
+        referenced by the inventory are valid and, consequently, read-only clients that
+        reference the inventory should continue to operate normally except for the brief moment
+        when the inventory is actually being updated. In practice, it is expected that upstream
+        web caching should be able to cover this momentary unavailability in the majority of 
+        cases. The following temporary names are defined:
+	</p>	
+        <ul>
+          <li>  
+          Temporary Version Directories: New versions should be created in a temporary version directory.
+          This could reside in the OCFL Storage Root workspace directory ("deposit"), the same location as
+          other version directories in an OCFL Object or elsewhere. In any case, only one such directory 
```

Comment:
what the does "the same location as other version directories in an OCFL Object or elsewhere" mean.

Reply:
I'm not saying where the workspace is....just that there has to be one and the contents are predictable and therefore identifiable.

Reply:
that i would change the sentence to say this, because right now i don't really know what that sentence says.

Reply:
I seems really hard to discuss a semi-blessed `deposit` directory inside objects in a PR. May I suggest creating a new issue (that points back to #320) that proposes guidance on this issue in the implementation notes while not changing the normative specification?

Note also should use code font `<code>deposit</code>` instead of quoting literals

Reply:
...this *is* guidance in the implementation notes

Reply:
Sorry for not being clear: my point is that it would be good to merge other parts of this PR but I don't think we can introduce a new in-object `deposit` directory without more discussion on a separate issue

Reply:
Limited discussion to "deposit" directory in Storage Root.